oci/spec_opts_windows.go

```diff
@@ -100,3 +100,32 @@ func escapeAndCombineArgs(args []string) string {
 	}
 	return strings.Join(escaped, " ")
 }
+
+// WithWindowsDevice adds a device exposed to a Windows (WCOW or LCOW) Container
+func WithWindowsDevice(IDType, ID string) SpecOpts {
```

Comment:
Yeah, I was looking back, and realised I'd copied `WithDevices` because the API (originally) matched, but that actually generates multiple devices for the supplied path.